Cargo.toml

```diff
@@ -20,7 +20,9 @@ exclude = [
 
 [features]
 reqwest-blocking-client = ["reqwest", "reqwest/blocking"]
-reqwest-client = ["reqwest", "opentelemetry/tokio-support"]
+reqwest-blocking-client-rustls = ["reqwest", "reqwest/rustls-tls", "reqwest/blocking"]
```

Comment:
Do we need `"reqwest/native-tls"` in the dependencies for `reqwest-blocking-client` as well?

Reply:
Oh sorry, my mistake, good spot! I had not tested that one admittedly.